clients/fides-js/src/components/ConsentModal.tsx

```diff
@@ -59,24 +58,39 @@ const ConsentModal = ({
 
   return (
     // @ts-ignore A11yDialog ref obj type isn't quite the same
-    <div data-testid="consent-modal" {...container} className="modal-container">
-      <div {...overlay} className="modal-overlay" />
-      <div data-testid="modal-content" {...dialog} className="modal-content">
+    <div
```

Comment:
this is all adding `fides-` to the beginning of every class name, but prettier adds new lines so the diff is more intense than it actually is!